pytorch_lightning/core/saving.py

```diff
@@ -52,10 +53,10 @@ def load_from_checkpoint(
         Primary way of loading a model from a checkpoint. When Lightning saves a checkpoint
         it stores the arguments passed to `__init__`  in the checkpoint under `module_arguments`
 
-        Any arguments specified through \*args and \*\*kwargs will override args stored in `module_arguments`.
+        Any arguments specified through \*args and \*\*kwargs will override args stored in `hparams`.
 
         Args:
-            checkpoint_path: Path to checkpoint.
+            checkpoint_path: Path to checkpoint. This can also be a URL.
```

Comment:
@williamFalcon what extra doc do you expect?

Reply:
oh cool. missed it